inc/namespace.php

```diff
@@ -174,6 +174,9 @@ function trace_wpdb_query( string $query, float $start_time, float $end_time, $e
  * @param array $trace
  */
 function send_trace_to_daemon( array $trace ) {
+	if ( function_exists( 'apply_filters' ) ) {
+		$trace = apply_filters( 'aws_xray.trace_to_daemon', $trace );
```

Comment:
Want to get in the habit of adding docblocks to filters so we can generate better docs down the line. Doesn't have to be super detailed right now.

Reply:
👍 added